home/modules/ROOT/pages/mobile.adoc

```diff
@@ -1,69 +1,100 @@
-:page-role: tiles
+= Couchbase Mobile 
+// :page-layout: landing-page-top-level-sdk
+// :page-role: tiles 
+:page-toclevels: 2
 :page-rank: 75
-:!sectids:
 
-include::partial$info-banner.adoc[]
+Couchbase Mobile brings the power of NoSQL JSON document database to the edge. 
+The combination of Sync Gateway and Couchbase Lite provides fast, efficient bidirectional synchronization of data between the edge and the cloud.
 
-= Couchbase Mobile
+This enables you to deploy offline-first mobile and edge applications with greater agility on premises or in any cloud. 
+Features include automatic conflict resolution, security, Full-Text Search, and SQL++ querying capabilities.
 
-++++
-<div class="card-row">
-++++
+== Components
 
-[.column]
-====== {empty}
-[.content]
-_Couchbase Mobile_ brings the power of NoSQL to the edge.
-The combination of _Sync Gateway_ and _Couchbase Lite_ coupled with the power of _Couchbase Server_ provides fast, efficient bidirectional synchronization of data between the edge and the cloud.
-Enabling you to deploy your offline-first mobile and embedded applications with greater agility on premises or in any cloud.
+Couchbase Mobile consists of:
 
-[.column]
-====== {empty}
-[.media-left]
-image::secure-manage-and-develop-with-ease.svg[,275]
+xref:couchbase-lite::introduction.adoc[Couchbase Lite] - Embedded NoSQL database for mobile and edge devices. 
+Couchbase Lite provides local storage, querying, and offline operations.
 
-++++
-</div>
-++++
+xref:sync-gateway::introduction.adoc[Sync Gateway] - Secure, high-performance synchronization server for cloud-to-edge data sync. 
+Provides authentication, access control, and RESTful API. 
+Both on-premises and Capella (Cloud) deployments use Sync Gateway.
 
-== Embedded Database on the Edge
-++++
-<div class="card-row two-column-row">
-++++
+xref:couchbase-edge-server:introduction:intro.adoc[Edge Server] - Lightweight database for edge/IoT environments. 
+Edge Server enables local data processing and synchronization with Couchbase Lite and Capella.
 
-[.column]
-.Couchbase Lite
+xref:cloud:app-services:index.adoc[App Services] - Fully managed synchronization service providing the same sync capabilities as Sync Gateway but without infrastructure management.
 
-[.content]
-Couchbase Lite is an embedded, NoSQL JSON Document Style database for your mobile apps.
-It natively supports all major operating systems and platforms.
-Its NoSQL client database provides CRUD, full-text search and query capabilities that runs locally on the device. +
-xref:couchbase-lite::introduction.adoc[Go to Couchbase Lite Docs]
 
-[.column]
-.Sync Gateway
+== Deployment Options
 
-[.content]
-Sync Gateway is an internet-facing synchronization mechanism designed to provide data synchronization for large-scale interactive web, mobile, and IoT applications. +
-xref:sync-gateway::introduction.adoc[Go to Sync Gateway Docs]
+=== Cloud (Capella)
+Components: Couchbase Lite + App Services
```

Comment:
NOTE: shout if you want to look at displaying the components as pills or similar in future.

Reply:
I surely will 😄